### PR TITLE
feat(accessibility): add visually hidden span with descriptive instru…

### PR DESCRIPTION
**Pull request by bvellek**
…ctions for screen reader use  a

In order to better support screen readers, a visually hidden span that instructs to arrow down as
you type for autocomplete suggestions. An aria-describedby attribute added to the input to direct to
the span description. a11y visually-hidden styles added to span to hide from sighted users.

issue #64

### CODE REVIEW

File: src/PlacesAutocomplete.js [hunk 1]
```diff
@@ -236,7 +236,12 @@ class PlacesAutocomplete extends Component {
         id="PlacesAutocomplete__root"
         style={this.inlineStyleFor('root')}
         className={this.classNameFor('root')}>
-        <input {...inputProps} />
+        <input {...inputProps} aria-describedby="PlacesAutocomplete__autocomplete-description"/>
+        <span 
+          id="PlacesAutocomplete__autocomplete-description"
+          style={this.inlineStyleFor('autocompleteDescription')}
```
Comment (kenny-hibino) [inline src/PlacesAutocomplete.js]: @bvellek We don't want regular (non-screen-reader) users to see this `span` tag. I think we should always use the default styles you defined in `defaultStyles.js` file. I don't see the use case where you need to override this style. What's your take on this?

Comment (bvellek) [inline src/PlacesAutocomplete.js]: @kenny-hibino I have been trying to think of any scenarios in which the user may want to see that span... but I haven't come up with any use cases either. I will submit another PR

**Comment by CoVoCre** [comment]
@hibiken what happened ??? 😢

**Comment by deastman** [comment]
@bvellek @sambokai Is it possible to describe what updates are needed to merge this PR. We're using this library and are eager to make it accessible via screen readers. If updates are needed to the previous PR, I know some folks who would be interested in working to get this PR in.

**Comment by bvellek** [comment]
hey @deastman i never received any response from the code owner at the time. My changes may no longer be relevant, but if you feel they are, feel free to copy the idea and submit a PR of your own

**Pull request closed without merging**